data/en/bots/495743974028476446.md

```diff
@@ -0,0 +1,47 @@
+---
+avatar: null
+cover: >-
+  https://images-ext-2.discordapp.net/external/lRYJygGawrXJiorjaxZvhTdwRxmmw1MXQgQnSCIJFlg/%3Fsize%3D1024/https/cdn.discordapp.com/avatars/495743974028476446/9d6bfa867ac5ee304f4c6f161826e62c.webp?width=473&height=473
+custom_path: psychic
+description: >-
+  Another general purpose but, this one has features that are actually kinda
+  cool, I think.
+github: null
+images: []
+link: >-
+  https://discordapp.com/api/oauth2/authorize?client_id=495743974028476446&permissions=8&scope=bot
+nsfw: false
+pagename: Psychic
+prefix: p*
+support: 'https://discord.gg/zEvdVEh'
+---
+# Psychic
+- Another general purpose but, this one has features that are actually kinda cool, I think.
+
+---
+> This code is majorly outdated compared to the current code used on the bot *RIGHT NOW*, please do not try hosting it as it's very much broken and I am 99% sure that it won't work, thanks!
+---
+
+# Psychic loves
+- [discord.py](https://github.com/Rapptz/discord.py/) ily bb <3
+
+# SERIOUS PROBLEMS
+- THE BOT CAN'T REMEMBER ANYTHING!! (NO DATABASES) THAT MEANS WE CAN'T MAKE COOL COMMANDS LIKE `p!warn <@user>` OR `p!mute <@user> [time] [reason]` ORRRRRR MAKE AN ECONOMY / LEVEL SYSTEM TO HELP MAKE THE BOT MORE APPEALING AND INTERESTING!!!!!
+
+# Introduction.
+- Hey, I am a bot made for handy utility and some nice commands to help your discord server, for example, we have a kick command and a ban command to help moderate your server. And also we have some fun commands to help spice up your server to not be that boring. 
+
+# Commands
+- There is a cool handy dandy [documentation page made on the bot](https://psychic.blizma.us) if you wish to see, woah isn't that *cool*.
```

Comment:
Link is broken

Reply:
Fixed.

Reply:
Is the DNS propagating now?
I'll test that again soon.

Reply:
Nah, the link was broken, I'm gonna buy a domain (for Psychic and not me) soon but I'm gonna use the default link.

Reply:
If you "fixed the link", how is there no new commit on this pull request?

Reply:
![not imgur](https://its-not-advertising-if-the-website-doesnt-exist-anymore.moustacheminer.com/chrome_2018-11-09_01-51-07.png)